src/extensions/core-api/stores.ts

```diff
@@ -20,3 +20,5 @@
  */
 
 export { ExtensionStore } from "../extension-store";
+export { MainIpcStore } from "../main-ipc-store";
```

Comment:
@Nokel81 somehow I missed this during reviews. Store is probably a wrong place for `Ipc` related things. Also class names indicate that these are stores (which they are not).

Reply:
Yes, I'll make a new PR to fix these names.